assignment5.txt

```diff
@@ -0,0 +1,11 @@
+Drew French
+
+1)	Passing by value: Passing a value to something.
+	Passing by reference: Passing the location where something is stored; arrays work like this.
+	Passing call-by-value: Calling a function & giving it values (such as sum(x,y)).
+
+2)	To end a function and give an output.
```

Comment:
Again, more detail. Returns give back control and data to the original calling function.